src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -50,21 +50,23 @@
    :render
    (fn [{:keys [props]}]
      (let [{:keys [color icon href disabled? onClick text style class-name]} props]
-       [:a {:className (or class-name "button")
-            :style (merge
-                    {:display "inline-flex" :alignItems "center" :justifyContent "center"
-                     :backgroundColor (if disabled? (:disabled-state style/colors) color)
-                     :cursor (when disabled? "default")
-                     :color (if disabled? (:text-light style/colors) "white")
-                     :fontWeight 500
-                     :minHeight 19 :minWidth 19
-                     :borderRadius 2 :padding (if text "0.7em 1em" "0.4em")
-                     :textDecoration "none"}
-                    (if (map? style) style {}))
-            :href (or href "javascript:;")
-            :onClick (if disabled? (create-error-message disabled?) onClick)
-            :onKeyDown (when (and onClick (not disabled?))
-                         (common/create-key-handler [:space] onClick))}
+       [:a (merge
```

Comment:
If we're going to ask developers to use this custom element instead of, say, `button`, we are obligated to support as much of the original control's API as feasible. Custom data properties, mouse hover events, etc. This is simply a convenient technique for doing that.